client/web/compose/src/views/Public/Pages/Records/View.vue

```diff
@@ -177,6 +177,11 @@ export default {
       type: Boolean,
       required: false,
     },
+
+    inEditMode: {
```

Comment:
Same here, just call it `edit`, since in certain edge cases, the prop will be unreliable.